mlx/coverity/coverity.py

```diff
@@ -150,30 +153,44 @@ def get_filtered_defects(self, node):
                     "rows": [list of dictionaries {"key": <key>, "value": <value>}]
                 }
         """
-        report_info("obtaining defects... ")
+        LOGGER.info("obtaining defects... ")
         column_names = set(node["col"])
         if "chart_attribute" in node and node["chart_attribute"].upper() in node.column_map:
             column_names.add(node["chart_attribute"])
         defects = self.coverity_service.get_defects(self.stream, node["filters"], column_names, self.snapshot)
-        report_info("%d received" % (defects["totalRows"]))
+        LOGGER.info("%d received" % (defects["totalRows"]))
         return defects
 
 
+def validate_coverity_credentials(app):
```

Comment:
```suggestion
def validate_coverity_credentials(config):
```
Better take the dict  `app.config.coverity_credentials` instead of the entire application object.